proto/viam/common/v1/common.proto

```diff
@@ -89,6 +89,14 @@ message Mesh {
   bytes mesh = 2;
 }
 
+message PointCloud {
+  bytes point_cloud = 1;
+}
+
+message Line {
+  repeated float segments = 1;
+}
```

Comment:
Do we still want to introduce the `Line` type if we are allowing `Transform` to have multiple `Geometry`s?

Reply:
I'm not sure what this has to do with Transform having multiple geometries

Reply:
should line be `bytes` too to match Pointcloud?

Reply:
Yeah, much easier to add it later than take it away.